unstructured_inference/yolox_functions.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) Megvii, Inc. and its affiliates.
+# Modifications copyright (C) 2023 by Unstructured
+
+import numpy as np
```

Comment:
For consistency with what we did for detectron, we should move `yolox` model code under `models/yolox.py`. There's also `inference/layout.py` but that was lifted from some old code in `unstructured` and may be due a refactor soon.

Reply:
I'd defer to @qued  on organization though since he's work with this code most closely of late.